extensions/wikia/CreateNewWiki/js/CreateNewWiki.js

```diff
@@ -275,6 +261,10 @@
 			this.answer = v;
 		},
 
+		onAuthSuccess: function () {
+			window.location.href = decodeURIComponent(this.redirectUrl).replace(/&/, '?');
```

Comment:
+1 to that, currently after succesful auth in force login modal we just continue the action that required logging in, without refreshing the page.